api/src/main/java/org/pmiops/workbench/cohortbuilder/util/QueryParameterValues.java

```diff
@@ -14,6 +17,18 @@
 public final class QueryParameterValues {
   private static final int MICROSECONDS_IN_MILLISECOND = 1000;
 
+  // For creating a Timestamp QueryParameterValue, use this formatter.
+  // example error when using the RowToInsert version (below): "Invalid format:
+  // "1989-02-17 00:00:00.000000" is too short".
+  public static final DateTimeFormatter QPV_TIMESTAMP_FORMATTER =
+      DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss.SSSSSSZZ").withZone(ZoneOffset.UTC);
+  // For parsing the string value of a timestamp QPV, use this formatter (with colon).
+  public static final DateTimeFormatter QPV_TIMESTAMP_TO_STRING_FORMATTER =
```

Comment:
this one is not used

Reply:
Yeah, I had a use for it but now I can't find it.